com.unity.formats.fbx/Documentation~/prefabs.md

```diff
@@ -85,21 +85,21 @@ When converting to an FBX Linked Prefab, the following window opens, displaying
 | __Object(s) Position__    | __Convert to FBX Linked Prefab__ always resets the root object's transform during export. However, the Prefab maintains the global transform for the root object. |
 | __Animated Skinned Mesh__ | Check this option to export animation on objects with skinned meshes.<br/><br/>If unchecked, the FBX Exporter does not export animation on skinned meshes. |
 | __Compatible Naming__     | Check this option to control renaming the GameObject and Materials during export. <br/><br/>The FBX Exporter ensures compatible naming with Autodesk® Maya® and Autodesk® Maya LT™ to avoid unexpected name changes between Unity and Autodesk® Maya® and Autodesk® Maya LT™. During export the FBX Exporter replaces characters in Unity names as follows:<br/> - Replaces invalid characters with underscores ("\_"). Invalid characters are all non-alphanumeric characters, except for colon (":").<br/> - Adds an underscore ("\_") to names that begin with a number. - Replaces diacritics. For example, replaces "é" with “e”.<br/><br/>**NOTE:** If you have a Material with a space in its name, the space is replaced with an underscore ("_"). This results in a new Material being created when it is imported. For example, the Material named "Default Material" is exported as "Default_Material" and is created as a new Material when it is imported. If you want the exported Material to match an existing Material in the scene, you must manually rename the Material before exporting. |
-| __Don't ask me again__    | Check this option to use the same **Convert Option** properties and hide this window when converting to FBX Linked Prefabs in the future. You can reset this option by turning on the **Show Convert UI** option under **Edit** > **Project Settings** > **Fbx Export** in Unity's top menu. |
+| __Don't ask me again__    | Check this option to use the same **Convert Option** properties and hide this window when converting to FBX Prefab Variants in the future. You can reset this option by turning on the **Show Convert UI** option under **Edit** > **Project Settings** > **Fbx Export** in Unity's top menu. |
 
 > **NOTE:** For FBX Model filenames, the FBX Exporter ensures that names do not contain invalid characters for the file system. The set of invalid characters may differ between file systems.
 
 
 
 <a name="conversion"></a>
-### Converting from Linked Prefabs to FBX Linked Prefabs
+### Converting from Prefab Variants to FBX Prefab Variants
```

Comment:
for the first change please keep "Linked Prefab", it is not the same as an FBX Linked Prefab